include/seqan/arg_parse/arg_parse_type_support.h

```diff
@@ -71,6 +93,23 @@ inline TTarget _cast(TString const s)
 }
 
 // ----------------------------------------------------------------------------
+// Function _castBool()
+// ----------------------------------------------------------------------------
+
+template <typename TString>
+inline bool _castBool(bool & dst, TString const s)
+{
+    // since the validValue check already verifies that the value if one of
```

Comment:
make a copy of TString and std::transform it to uppercase so we have the lower and mixed cases, as well!